Dockerfile.buildrpm

```diff
@@ -1,10 +1,39 @@
-FROM welder/web-nodejs:latest
+FROM fedora:latest
 LABEL maintainer="Xiaofeng Wang" \
       email="xiaofwan@redhat.com" \
       baseimage="Fedora:latest" \
       description="A welder-web RPM builder container running on Fedora"
-# work around https://bugzilla.redhat.com/show_bug.cgi?id=1532293 by upgrading cmake first
-RUN dnf upgrade -y cmake && dnf install -y rpm-build jq && dnf clean all
+
+RUN dnf install -y make cmake rpm-build gnupg tar xz curl jq && dnf clean all
+
+RUN echo 'PATH=/usr/local/bin/:$PATH' >> /etc/bashrc
+
+# Based on official node docker image
+# gpg keys listed at https://github.com/nodejs/node
+RUN set -ex \
```

Comment:
Don't like the repetition of a huge block of code between 3 docker files.

I think the way to go here would be to :+1: 

1) build the web image as first thing in the shared target
2) base Dockerfile.buildrpm on welder/web image and change Makefile target to depend on the shared target
3) Same goes for the test image, better not duplicate the Nodejs configuration part.

Reply:
Or copy nodejs from Travis to container. But that'll not work in downstream Jenkins test.

Reply:
Errr nope, don't copy anything from an Ubuntu container to Fedora. That's asking for trouble.